### PR TITLE
F-81B RVF test fixes

### PR DESCRIPTION
**Pull request by apoorva-rao**
Signed-off-by: Apoorva Rao <apoorva.d.rao@seagate.com>

# Problem Statement
- Test fixes for F-81B (simple object delete/overwrite) test cases as a part of 
- [CORTX-33494](https://jts.seagate.com/browse/CORTX-33494)
- [CORTX-33496](https://jts.seagate.com/browse/CORTX-33496)

# Design
- https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/1130233857/TDS+F-81B+S3+API+Object+CRUD+Delete+Async+Delete+Simple+Object

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [x] New/Affected tests are executed on Latest Build
-  [x] Attach test execution logs
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [x] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by archanabpatil — CHANGES_REQUESTED** [review]

**Comment by archanabpatil** [comment]
execution logs link ?

File: libs/s3/s3_common_test_lib.py [hunk 1]
```diff
@@ -99,6 +101,40 @@ def get_cortx_capacity() -> tuple:
...
+def get_cortx_rgw_bytecount() -> tuple:
+    """Get bytecount stats for CORTX RGW."""
+    csm_obj = csm_api_factory("rest")
+    LOG.info("Sleep for capacity update interval %s seconds", const.Rest.CAPACITY_UPDATE_TIMEOUT)
+    time.sleep(const.Rest.CAPACITY_UPDATE_TIMEOUT)
+    response = csm_obj.verify_get_bytecount()
+    return response['bytecount']
+
+
+def poll_cluster_capacity(check_increase=True, sleep_interval=300, max_retries=5,
```
[the remark this answers is not in the document]
Comment (apoorva-rao) [reply libs/s3/s3_common_test_lib.py:113]: For our tests we are retrying 3 times as there is an added capacity update timeout of 10 minutes. Kept the max_retries default as 5 minutes, as the update timeout is expected to decrease and we need to have sufficient time to allow GC to complete it's processing. We have requested for recommended time estimates and for now we have added our own estimates based on manual verification.

File: tests/s3/test_async_delete.py [hunk 2]
```diff
@@ -138,49 +139,56 @@ def test_random_sized_object_delete_and_overwrite_44815(self):
...
-            resp = self.s3_test_obj.put_object(self.bucket_name, obj_name, fpath)
-            assert_utils.assert_true(resp[0], resp[1])
-        overwrite_file_size_1 = os.path.getsize(self.upload_paths[1])
-        used_capacity1 = get_cortx_capacity()[-1]
-        self.log.info("CORTX capacity after upload: %s", used_capacity1)
-        self.log.info("Step 2: Delete object 1")
-        resp = self.s3_test_obj.delete_object(self.bucket_name, self.obj_names[0])
+        self.log.info("Step 1: Upload 2 objects having random size")
+        create_random_file(self.upload_paths[0], DEL_CFG["random_obj_size"]["min"],
+                            DEL_CFG["random_obj_size"]["max"])
```
[the remark this answers is not in the document]
Comment (apoorva-rao) [reply tests/s3/test_async_delete.py]: fixed this and updated the PR

**Review by kachhwahadivya — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

**Comment by apoorva-rao** [comment]
> execution logs link ?

Logs attached with the Jira tickets - [CORTX-33494](https://jts.seagate.com/browse/CORTX-33494) and [CORTX-33496](https://jts.seagate.com/browse/CORTX-33496)

**Review by archanabpatil — APPROVED** [review]

**Review by sonalk0209 — APPROVED** [review]
Approving for merge!!!!

**Comment by apoorva-rao** [comment]
Pytest collection verified - 
======================== 3218 tests collected in 17.02s ========================

**Pull request merged**